apps/src/templates/RegionalPartnerSearch.jsx

```diff
@@ -52,7 +52,7 @@ class RegionalPartnerSearch extends Component {
 
     if (partnerId) {
       if (partnerId === "0") {
-        showZip = false;
+        showZip = true;
```

Comment:
Do we ever not show the zip now? Is there still a need for this state variable?

Reply:
Good question.  We still hide it while the async retrieval of a partner (provided by URL parameter) is done, which is a nice UX touch.